Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -108,12 +108,16 @@ - (void)viewDidLoad
 }
 
 - (void)setOptions:(WPMediaPickerOptions *)options {
+    WPMediaPickerOptions *originalOptions = _options;
     _options = [options copy];
+    BOOL refreshNeeded = (originalOptions.filter != options.filter) | (originalOptions.showMostRecentFirst != options.showMostRecentFirst) | (originalOptions.allowCaptureOfMedia != options.allowCaptureOfMedia);
```

Comment:
We could implement `isEqual` on `WPMediaPickerOptions` and then just do `if (_options != options)`